cypress-tests/cypress/constants/selectors/common.js

```diff
@@ -166,7 +166,7 @@ export const commonSelectors = {
   resetPasswordPageDescription: '[data-cy="reset-password-page-description"]',
   backToLoginButton: '[data-cy="back-to-login"]',
   breadcrumbTitle: '[data-cy="app-header-label"]>>',
-  breadcrumbPageTitle: '[data-cy="breadcrumb-page-title"]',
+  breadcrumbPageTitle: '[data-cy="app-header-label"]',
```

Comment:
Create a new selector if needed, don't update the existing selectors it will break other test cases.